apps/web/src/pages/auth/InvitationPage.tsx

```diff
@@ -45,7 +45,7 @@ export default function InvitationPage() {
   useEffect(() => {
     // auto accept invitation when logged in as invited user
     if (isLoggedInAsInvitedUser) {
-      submitToken(tokensRef.current.token as string, tokensRef.current.invitationToken as string, true);
+      submitToken(tokensRef.current.token as string, tokensRef.current.invitationToken as string, true, false);
```

Comment:
it's hard to understand what are there `true/false` arguments, please change it to the object, which will improve readability